packages/devtools_app/lib/src/flutter/status_line.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../devtools.dart' as devtools;
+
+/// The status line widget displayed at the bottom of DevTools.
+class StatusLine extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // TODO(devoncarew): Break this into an isolates area, a connection status
+    // area, a page specific area, and a documentation link area.
+
+    final textTheme = Theme.of(context).textTheme;
+
+    return Container(
+      height: 24.0,
```

Comment:
is specifying the height necessary? if not we can remove the Container widget entirely

Reply:
I'm seeing a lot of 24.0 constants hardcoded in devtools. It would be nice to add a default line height constant to the theme.

Reply:
I'd like to make sure we don't overflow the vertical space here. I also specify the height on the containing widget (in`scaffold.dart`) as w/o that I hit an assert in our `Split` widget.